src/routes/settings/dhcp.rs

```diff
@@ -80,6 +82,7 @@ pub fn put_dhcp(env: State<Env>, _auth: User, data: Json<DhcpSettings>) -> Reply
     SetupVarsEntry::DhcpLeasetime.write(&settings.lease_time.to_string(), &env)?;
     SetupVarsEntry::PiholeDomain.write(&settings.domain, &env)?;
     SetupVarsEntry::DhcpIpv6.write(&settings.ipv6_support.to_string(), &env)?;
+    SetupVarsEntry::DhcpRapidCommit.write(&settings.rapid_commit.to_string(), &env)?;
```

Comment:
How does it know that the corresponding setting is `DHCP_rapid_commit`? I only see this string appear in the tests.

Reply:
Here is where the enums variants are mapped to their setting keys:
https://github.com/pi-hole/api/blob/1d0dacaaf59b0035ebf728b89daaa50414243ff4/src/settings/entries.rs#L220